src/content/self-hosted/eks.mdx

```diff
@@ -83,6 +83,12 @@ license: 1234567890ABCD==
 subdomain: dev.example.com
 cluster:
   endpoint: "https://XXXXXX.gr7.us-west-2.eks.amazonaws.com"
+
+ingress-nginx:
+  controller:
+    service:
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-type: nlb
```

Comment:
Could we add a short explanation about what this annotation does, and why NLB is preferred over ELB?